iothub/service/src/ServiceClient.cs

```diff
@@ -205,5 +205,24 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
         /// <param name="message">The message containing the notification</param>
         /// <returns></returns>
         public abstract Task SendAsync(string deviceId, string moduleId, Message message);
+
+        /// <summary>
+        /// Initiates a new cloud-to-device stream.
+        /// </summary>
+        /// <param name="deviceId">Device Id</param>
+        /// <param name="deviceStreamRequest">Configuration needed for initiating a cloud-to-device stream.</param>
+        /// <param name="cancellationToken">Token used for controlling the termination of the asynchronous call.</param>
+        /// <returns>The result of the cloud-to-device stream request, or null of the request itself could not be completed.</returns>
+        public abstract Task<DeviceStreamResponse> CreateStreamAsync(string deviceId, DeviceStreamRequest deviceStreamRequest, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Initiates a new cloud-to-device stream.
+        /// </summary>
+        /// <param name="deviceId">Device Id</param>
+        /// <param name="moduleId">Module ID</param>
```

Comment:
device Id and module Id could have a bit more helpful descriptions here and above. For instance: "The Id of the device to create the stream to"

Reply:
I figure these are just being copied from the old branch, but we could spruce it up a bit